### ISSUE TITLE
Yoast is not analyzing the ACF fields

### ISSUE THREAD

**Issue by havn** [Yoast/yoast-acf-analysis#302]
* [ x] I've read and understood the [contribution guidelines](https://github.com/Yoast/yoast-acf-analysis/blob/develop/.github/CONTRIBUTING.md).
* [ x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened.
I have the issue that Yoast is not analyzing the ACF fields correctly, it's like it doesn't evcen see ACF fields. It's not reporting the right word count (not even close), it also misses image alt tags, h2 and h3 headers, and some other issues.

### Please describe what you expected to happen and why.
I expect Yoast to give a proper analysis of pages with ACF fields. (Including word count, image alt tags, h2 and h3 headers etcetera)

### How can we reproduce this behavior?
This is just one example:
Page: [https://www.mindfulness-op-maat.nl/wp/anja/](https://www.mindfulness-op-maat.nl/wp/anja/)
This page contains over 500 words, yet Yoast gives a red bullet and reports:
“Text length: The text contains 89 words. This is far below the recommended minimum of 300 words. Add more content.”

### Technical info
* WordPress version:  5.8.1
* ACF Content Analysis for Yoast SEO version: 3.0.1
* Yoast SEO version:  17.5
* ACF type: free
* ACF version: 5.10.2
* Relevant plugins in case of a bug: - (maybe Elementor Pro?)

### What I tried to resolve the issue:
– turning off all plugins except Yoast and ACF content analysis / no effect
– reverting to a wordpress default theme (like twentyseventeen) / no effect
– reïnstalling both Yoast and ACF content analysis / no effect
- a.f.a.i.c.t there are no plugin conflicts

Any pointers or tips would be greatly appreciated, thanks!

**Comment by growill** [comment]
I have the same issue on several websites...
WordPress 5.8.2
Yoast SEO 17.6
ACF Pro 5.11.1
ACF Content Analysis for Yoast SEO 3.0.1 

No builder.

I someone know how to fix the problem It would be greatly appreciated! 
Thanks !

**Comment by gypsytales** [comment]
I just installed the plugin and also having issues. It's not seeing internal links at all even thought there are several.

**Comment by madebyiando** [comment]
I'm have the same issue, no external links are being detected. The plugin doesn't seem to have been updated in a good while, are you still supporting it?

**Comment by RawMajkel** [comment]
It is still not fixed? Wow. People (me included) were talking about this bug back in 2017.
https://github.com/Yoast/wordpress-seo/issues/7441

Maybe this plugin is just dead?

**Comment by mgcross** [comment]
Same issue with WP 6..1, Yoast 19.9, ACF Content Analysis for Yoast SEO  3.0.1. Looks like this has not been updated in any substantial way since 2020.

**Comment by yebsuadp** [comment]
Same issue here.
Any solutions since Nov. 2022?

**Comment by ariskuckovic-abtion** [comment]
This is sad - no replies from developers here OR on the WP support-page.

**Comment by RawMajkel** [comment]
> This is sad - no replies from developers here OR on the WP support-page.

Yes, it is. Especially since I believe it's not that hard to implement correctly. Maybe it's time to admit that the project is no longer supported?

**Comment by ariskuckovic-abtion** [comment]
I'm looking into a solution for my clients as we speak.
This plugin will be deleted from my sites - makes no sense to keep it around, as it looks abandoned.

**Comment by tahmidbintaslim** [comment]
@yoastbot Bump

**Comment by hookson** [comment]
Adding to the posts here in the hope it bumps this bug up the priority list @yoastbot 

Have tested with clean WP install:
WordPress 6.7.1
ACF Pro 6.3.11
Yoast SEO* 24.0
ACF Content Analysis for Yoast SEO 3.1
Twenty Twenty-Five theme

External and Internal links in URL and Text fields are not picked up the SEO analysis ('No outbound links appear in this page.' and 'No internal links appear in this page')

Those added in WYSIWYG fields are recognized.

*Tested with Yoast Free and Premium and both have the same issue.

**Comment by masterfugazi** [comment]
I'm having this exact same issue...as above. Would be great to get this resolved....
